### DISCUSSION TITLE
Class 'Phalcon\Version' not found in C:\wamp64\www\phal\vendor\phalcon\devtools\bootstrap\autoload.php:134

### DISCUSSION THREAD

**Discussion by imakepossible** [phalcon/phalcon#114] [General]
Hi,

i have installed Phalcon version 5.0.0alpha3 on window and it is showing in phpinfo and in php --m 
but when i run phalcon on command line, i am getting error

Fatal error: Uncaught Error: Class 'Phalcon\Version' not found in C:\wamp64\www\phal\vendor\phalcon\devtools\bootstrap\autoload.php:134
Stack trace:
#0 C:\wamp64\www\phal\vendor\phalcon\devtools\phalcon(46): require()
#1 {main}
  thrown in C:\wamp64\www\phal\vendor\phalcon\devtools\bootstrap\autoload.php on line 134

**Comment by niden** [comment]
The `Version` component has been moved in the `Support` namespace and is no longer a static method.

```php
echo (new Phalcon\Support\Version())->get();  
```
